### PR TITLE
[DAG notification] ping for unseen reports

### PR DESCRIPTION
**Pull request by Pierlou**

### CODE REVIEW

File: dgv/monitoring/DAG-notification.py [hunk 1]
```diff
@@ -511,6 +548,11 @@ def publish_mattermost(ti):
         python_callable=alert_if_awaiting_spam_comments,
     )
 
+    alert_if_awaiting_spam_comments = PythonOperator(
```
Comment (maudetes) [inline dgv/monitoring/DAG-notification.py:551]: What was that not here?

File: dgv/monitoring/DAG-notification.py [hunk 2]
```diff
@@ -183,6 +183,43 @@ def alert_if_awaiting_spam_comments():
         send_message(message, MATTERMOST_MODERATION_NOUVEAUTES)
 
 
+def alert_if_new_reports():
+    # DAG runs every 5min but if it fails we catch up with this variable
+    previous_report_check = Variable.get(
```
Comment (maudetes) [inline dgv/monitoring/DAG-notification.py:188]: Can't we fetch latest run instead of using variables?

**Review by maudetes — APPROVED** [review]
Looks good to me!

File: dgv/monitoring/DAG-notification.py (hunk 1 above)
Comment (Pierlou) [reply dgv/monitoring/DAG-notification.py]: Forgot to modify after copy/paste 🙃 thanks for the alert

File: dgv/monitoring/DAG-notification.py (hunk 2 above)
Comment (Pierlou) [reply dgv/monitoring/DAG-notification.py:188]: That's more complex, getting the latest run in which this specific task didn't fail. I can try if needed, but this should also work

**Pull request merged**